curator-x-discovery/src/main/java/org/apache/curator/x/discovery/ServiceCacheBuilder.java

```diff
@@ -46,6 +46,7 @@
      * @param threadFactory factory
      * @return this
      */
```

Comment:
Please add to the Javadoc:
`* @deprecated use {@link #executorService(ExecutorService)} instead`

Reply:
done

Reply:
Sorry if I wasn't clear, but I didn't mean to drop the `@Deprecated`. `@Deprecated` and `@deprecated` should be used together. Please refer to:
https://docs.oracle.com/javase/8/docs/technotes/guides/javadoc/deprecation/deprecation.html
Their conclusion is "Thus, using the `@Deprecated` annotation to generate warnings is more portable that relying on the `@deprecated` Javadoc tag."

Same for `ServiceProviderBuilder.threadFactory(...)`.

Reply:
Also, it turns out that deprecation of interface method doesn't affect overriding methods, as said in the same link:
"It is possible for a non-deprecated property to hide or override a deprecated one, which removes deprecation. As developer of an API, it is your responsibility to deprecate overrides of a deprecated method, if in fact they should be deprecated."
Could you deprecate also the two `threadFactory(...)` overriding methods?

Reply:
done.